dns/types/records/PTR.nix

```
@@ -0,0 +1,25 @@
+#
+# © 2019 Kirill Elagin <kirelagin@gmail.com>
+#
+# SPDX-License-Identifier: MIT
+#
+
+{ pkgs }:
+
+let
+  inherit (pkgs.lib) mkOption types;
+
+in
+
+{
+  rtype = "PTR";
+  options = {
+    cname = mkOption {
```

I’m trying to follow the field names from the RFC, so:

```suggestion
    ptrdname = mkOption {
```